server/core/src/email.rs

```diff
@@ -59,7 +70,7 @@ where
     <T as AsyncTransport>::Error: std::fmt::Debug,
 {
     let to = to.parse().expect("to should be a valid email address");
-    let from = "Veryrezsi <noreply@veryrezsi.com>"
+    let from = "Veryrezsi <noreply@dev.veryrezsi.cloud>"

```

Comment:
Should this address be configurable?

Reply:
I will record it as an implementation detail!

Reply:
Thanks